src/act.core.etl/Gatherer.cs

```diff
@@ -509,7 +509,9 @@ public async Task<int> NotifyUnassignedNodes()
                 foreach (var node in nodes)
                 {
                     var name = node.Owner.OwnerText(false);
-                    var email = node.Owner.Email;
+                    var email = !string.IsNullOrEmpty(node.RemedyGroupEmailList)
+                        ? new[] {node.Owner.Email, node.RemedyGroupEmailList}
+                        : new[] {node.Owner.Email};
```

Comment:
I think in all these cases where we are adding this remedy email list, I would either use a List and just add it if it is not empty or as an alternative change the SendEmail method to ignore empty/null emails.

Reply:
I updated SendEmail method to ignore empty/null emails.